ui/app/AppLayouts/Chat/ChatColumn/ChatComponents/ChatCommandButton.qml

```diff
@@ -61,6 +61,8 @@ Rectangle {
     MouseArea {
         anchors.fill: parent
         cursorShape: Qt.PointingHandCursor
-        onClicked: root.onClicked
+        onClicked: {
```

Comment:
just being curious,  is there a difference between using `root.onClicked` directly and putting it inside a `{}` block?

Reply:
I'm not 100% to be honest. Sometimes it doesn't work well. I assume here I just changed it because I had put a `console.log` and removed it. Oh well.